tests/test_config.py

```diff
@@ -320,3 +321,25 @@ def test_ws_max_size():
     config = Config(app=asgi_app, ws_max_size=1000)
     config.load()
     assert config.ws_max_size == 1000
+
+
+@pytest.mark.parametrize(
+    "reload, workers",
+    [
+        (True, 1),
+        (False, 2),
+    ],
+    ids=["--reload=True --workers=1", "--reload=False --workers=2"],
+)
+@pytest.mark.skipif(sys.platform == "win32", reason="require unix-like system")
+def test_bind_unix_socket_works_with_reload_or_workers(tmp_path, reload, workers):
+    uds_file = tmp_path / "uvicorn.sock"
+    config = Config(
+        app=asgi_app, uds=uds_file.as_posix(), reload=reload, workers=workers
```

Comment:
Is there a reason for not adding any test for `fd`?

Reply:
no, I added one